app/javascript/components/supplemental_application/sections/preferences/AntiDisplacementHousingPanel.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Select } from 'react-form'
+
+import FormGrid  from '~/components/molecules/FormGrid'
+import { buildHouseholdMembersOptions } from '~/components/applications/application_form/preferences/utils.js'
+import { FormItem, Comment, statusOptions } from './utils'
+import formOptions from '~/components/applications/application_form/formOptions'
+
+const {
+  preference_proof_options_default
+} = formOptions
+
+export const AntiDisplacementHousingPanel = ({ data, applicationMembers }) => {
+  const applicationMembersOptions = buildHouseholdMembersOptions(applicationMembers)
+  return (
+    <React.Fragment>
+    <FormGrid.Row expand={false}>
+      <FormItem label="Preference Name">
+        <div className="text-value">
+          Anti-Displacement Housing Preference
```

Comment:
@fddayan Is it possible to use the `getPreferenceName` function in these preference panels to provide the value for the preference name?

Reply:
I do not think  what `getPreferenceName` returns maps exactly to what the screen in the story is showing. if you check the values return in `getPreferenceName` and the panel names there are some differences. 
`Rent Burdened Preference` vs `Rent Burdened Housing Preference`
 `Live in San Francisco Preference` vs `Live or Work in San Francisco`
whatever I'm showing the panel is exactly what the screen in the story is showing. Not sure if this a problem in the screens and we should actually render the `preference_name`, that would be question to jesse. I would leave it as it is and wait for jesse. Making the preference name in the row and the panel might be intentional.

Reply:
yes you're right they're slightly different, looks like that's a question for jesse, let's leave as you have it for now since it does match the current designs 👍